src/games/strategy/triplea/ai/fastAI/FastOddsEstimator.java

```diff
@@ -0,0 +1,146 @@
+package games.strategy.triplea.ai.fastAI;
```

Comment:
Point taken, I've measured the clone data taking over 2s. Seems like the clone data gets slower proportionate to the overall game size.

Why is the casualty sorting so slow? Is it slow when game data is large, or just slow when there are lots of units?